.circleci/config.yml

```diff
@@ -37,11 +38,53 @@ jobs:
           key: v1-dependencies-{{ .Environment.CIRCLE_JOB }}-{{ arch }}-{{ checksum "package.json" }}
           paths:
             - node_modules
+  build:
+    docker:
+      - image: docker:stable
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          command: |
+            docker build \
+              --no-cache \
+              -t coredump/flamebot:latest \
+              -t coredump/flamebot:${CIRCLE_BRANCH} \
+              .
+      - deploy:
+          name: Push Docker image
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_API_KEY && \
+            docker push coredump/flamebot
 
 workflows:
   version: 2
-  build:
+
+  # Build on push
+  push:
     jobs:
-      - build-node8
-      - build-node10
+      - test-node8
+      - test-node10
       - lint
+      - build:
+          requires:
+            - test-node8
+            - test-node10
+            - lint
+          filters:
+            branches:
+              only:
+                - master
+
+  # Build every week on Monday at 04:00 am
```

Comment:
What's the point in that? Getting updates from the base image?

Reply:
Exactly :)